Gemfile

```diff
@@ -3,7 +3,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
```

Comment:
I'd like to add an explicit dependency on the 'responders' gem. From the ([release notes](https://guides.rubyonrails.org/4_2_release_notes.html#respond-with-class-level-respond-to)):
> `respond_with` and the corresponding class-level `respond_to` have been moved to the [responders](https://github.com/plataformatec/responders) gem
We use respond_with, but are saved because devise happens to depend on responders. Better to have explicit dependency.

Reply:
Sounds good to me. Feel free to just commit here; I'm sure @bagedevimo is fine with it.